CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Added support for self-service CSS and brand images.
 - Added support for carousel_play_speed and carousel_fade_time configs.
 - Added support to toggle on cloudsearch via Meta > cloudsearch feature toggle.
+- Add translations for live events
```

Comment:
I think you need to merge the latest from develop into your branch... this block of stuff has been released under 1.4.0

Reply:
mybad moved line to unreleased